src/config/dynamic-theme-fixes.config

```diff
@@ -3261,6 +3261,10 @@ a.external::after
 ul.main-menu .top-level-entry::before
 
 CSS
+.copy-icon {
+    background: var(--darkreader-neutral-text) url(/static/media/clippy.92fffda9.svg) 50% no-repeat !important;
+    border: 1px solid var(--darkreader-neutral-text) !important;
```

Comment:
Why coloring it so bright? The current color looks fine for me.

Reply:
The documentation about **[dynamic variables](https://github.com/darkreader/darkreader/blob/master/CONTRIBUTING.md#dynamic-variables)** made me think it was recommended to use them instead of hardcoded colors.